Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/Model/Building/SqlStorageTypeInformationProvider.cs

```diff
@@ -278,16 +277,16 @@ private TypeConverter GetTypeConverter (Type nullableValueType, Type underlyingT
           new AdvancedEnumConverter(enumType));
     }
 
-    private string GetStorageTypeStringForVarType (string varType, int? maxLength)
+    private string GetStorageTypeStringForVarType (string varType, int maxLength)
```

Comment:
Should the bugfix commit be in its own pull request? If so, target branch "develop" or our "quest" branch RM-7194?

Reply:
yes, dedicated PR. it can go into develop or be part of the PRs n the quest branch. either way, we wil integrate those anyway as individual merges on develop so it's only a question of when it will show up in the next build. I'm okay either way.